src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -1294,6 +1294,9 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 
 									if (dynamicallyAccessedMemberNode.DynamicallyAccessedMemberTypes.HasFlag (DynamicallyAccessedMemberTypes.PublicProperties))
 										propagatedMemberTypes |= DynamicallyAccessedMemberTypes.PublicProperties;
+
+									if (dynamicallyAccessedMemberNode.DynamicallyAccessedMemberTypes.HasFlag (DynamicallyAccessedMemberTypes.Interfaces))
+										propagatedMemberTypes |= DynamicallyAccessedMemberTypes.Interfaces;
```

Comment:
Consider creating a mask here and &'ing, instead of checking each flag individually.

Reply:
Also, whether inclusion of exclusion is preferred, i.e. if we forget to update this, would we prefer to propagate more or less?